From 5bd3bf59831b508ee378ed4c105819e7c580171b Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 10 Jun 2020 21:37:21 +0800
Subject: [PATCH] f2fs: sync upstream for avoid gcc warning

fs/f2fs/inode.c: In function 'sanity_check_inode':
warning: format '%lu' expects argument of type 'long unsigned int',
but argument 6 has type 'blkcnt_t {aka long long unsigned int}'

Change-Id: Ibc600242dbdc600dd31111b5ba1413c329e26c5e
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 fs/f2fs/inode.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fs/f2fs/inode.c b/fs/f2fs/inode.c
index be6ac33461d1..0c9e62e4c1ec 100644
--- a/fs/f2fs/inode.c
+++ b/fs/f2fs/inode.c
@@ -301,10 +301,10 @@ static bool sanity_check_inode(struct inode *inode, struct page *node_page)
 		if (le64_to_cpu(ri->i_compr_blocks) >
 				SECTOR_TO_BLOCK(inode->i_blocks)) {
 			f2fs_warn(sbi, "%s: inode (ino=%lx) has inconsistent "
-				"i_compr_blocks:%llu, i_blocks:%lu, run fsck to fix",
+				"i_compr_blocks:%llu, i_blocks:%llu, run fsck to fix",
 				  __func__, inode->i_ino,
 				  le64_to_cpu(ri->i_compr_blocks),
-				  SECTOR_TO_BLOCK(inode->i_blocks));
+				  (u64)SECTOR_TO_BLOCK(inode->i_blocks));
 			return false;
 		}
 		if (ri->i_log_cluster_size < MIN_COMPRESS_LOG_SIZE ||
-- 
2.35.3

